src/constants.js

```diff
@@ -250,3 +250,9 @@ export const PLACEHOLDER_USER = { type: 'user', id: '2', name: '' };
 
 /* ------------------ Integrations ------------------------- */
 export const APP_INTEGRATION = 'app_integration';
+
+/* ------------------ Status Codes ------------------------- */
+export const CONFLICT_CODE = 409;
```

Comment:
There is already a section for this above on line 240.

Reply:
+ naming convention should be inverted